gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=29.69.0
+version=29.69.1
```

Comment:
We could bump the minor version instead of the patch, but since this is just a bug fix and not adding any new APIs I thought it could be a patch bump. @reviewers Let me know if you disagree